PerimeterXModule/PxModule.cs

```diff
@@ -454,11 +486,8 @@ private void HandleVerification(HttpApplication application)
             }
             else // No custom verification handler -> continue regular flow
             {
-                if (!verified && config.MonitorMode == false)
-                {
-                    BlockRequest(pxContext, config);
-                    application.CompleteRequest();
-                }
+                BlockRequest(pxContext, config);
+                application.CompleteRequest();
```

Comment:
Unless I'm missing something, it looks like we go to `BlockRequest` and `CompleteRequest` (instead of letting it pass) on _any_ case - both `verified` and `!verified`?